readthedocs/settings/base.py

```diff
@@ -753,6 +753,15 @@ def SOCIALACCOUNT_PROVIDERS(self):
         "signup": "readthedocs.forms.SignupFormWithNewsletter",
     }
 
+    GITHUB_APP_ID = 1234
+    GITHUB_APP_NAME = "readthedocs"
+    GITHUB_APP_PRIVATE_KEY = ""
+    GITHUB_APP_WEBHOOK_SECRET = ""
+
+    @property
+    def GITHUB_APP_CLIENT_ID(self):
+        return self.SOCIALACCOUNT_PROVIDERS["githubapp"]["APPS"][0]["client_id"]
```

Comment:
Can we consolidate patterns here, there are some inconsistencies forming here between `SOCIALACCOUNT_PROVIDERS_GITHUBAPP_*` and `GITHUB_APP_*`. I feel these make more sense all under the existing pattern.

Reply:
We don't have settings named `SOCIALACCOUNT_PROVIDERS_GITHUBAPP_*`, they are all set in the `SOCIALACCOUNT_PROVIDERS` setting, which is for allauth, while the `GITHUB_APP_*` settings are for interacting with the github app itself.

Reply:
We have `SOCIALACCOUNT_PROVIDERS_GITHUBAPP_*` env variables that feed settings. It's confusing to have all other providers on one pattern and the GHA on a different pattern.

Reply:
the github app settings aren't related to oauth/social account, I think it makes sense for them to be separate, also we shouldn't just add more attributes in the allauth settings that aren't used there.